script/add-payment-partials-to-marriage-abroad-services-data.rb

```diff
@@ -0,0 +1,52 @@
+countries = %w(
+  belgium
+  china
+  ecuador
+  egypt
+  finland
+  lebanon
+  macao
+  mongolia
+  morocco
+  norway
+  peru
+  philippines
+  qatar
+  south-korea
+  thailand
+  turkey
+  united-arab-emirates
+  vietnam
+)
+
+payment_method_partial = {
+  'default'     => 'pay_by_cash_or_credit_card_no_cheque',
+  'finland'     => 'pay_in_euros_or_visa_electron',
+  'norway'      => 'pay_by_visas_or_mastercard',
+  'philippines' => 'pay_by_cash_or_bank_cheque',
+  'south-korea' => 'pay_by_cash_in_local_currency_or_credit_card_not_cheque'
+}
+
+marriage_abroad_services_file = Rails.root.join('lib', 'data', 'marriage_abroad_services.yml')
+marriage_abroad_services = YAML.load_file(marriage_abroad_services_file)
+
+countries.each do |country|
+  payment_partial_name = payment_method_partial[country] ||
+    payment_method_partial['default']
```

Comment:
> What's the purpose of this default value? It doesn't look as if it's being used in the app code.

@floehopper: It's used on this line in the temporary script. It's the default value stored in the marriage_abroad_services.yml file when no country specific value exists.